editor/block-mover/test/mover-label.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getBlockMoverLabel, getMultiBlockMoverLabel } from '../mover-label';
+
+describe( 'block mover', () => {
+	const dir_up = -1,
```

Comment:
> My worry is: Why is this relevant to the tests only, and not runtime usage?

For runtime usage, it's only invoked from two statements in `block-mover/index.js` - the only reason I added these variables was to avoid the repetition of literals specifically within the tests. You're right though, when other arguments are still static it feels a bit odd.

I'm not massively wedded to it at all and am more than happy just to submit another PR removing them.

You might be onto something with the direction constants though — I could, however, make them static properties of the `getBlockMoverLabel` function:

```
function getBlockMoverLabel() { ... }

getBlockMoverLabel.DIRECTION_UP = -1;
getBlockMoverLabel.DIRECTION_DOWN = 1;

export { getBlockMoverLabel };
```

Happy to do something like that if you'd like, assuming there's no issue with the constant syntax suggested above.